src/LightStep/LightStep.csproj

```diff
@@ -3,18 +3,23 @@
     <AssemblyName>LightStep</AssemblyName>
     <RootNamespace>LightStep</RootNamespace>
     <LangVersion>7.0</LangVersion>
-    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Google.Api.CommonProtos" Version="1.3.0" />
-    <PackageReference Include="Google.Protobuf" Version="3.12.4" />
-    <PackageReference Include="LibLog" Version="5.0.3">
+    <PackageReference Include="Google.Api.CommonProtos" Version="2.1.0" />
+    <PackageReference Include="Google.Protobuf" Version="3.13.0" />
+    <PackageReference Include="LibLog" Version="5.0.8">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="OpenTracing" Version="0.12.1" />
-    <PackageReference Include="System.Json" Version="4.5.0" />
+    <PackageReference Include="System.Json" Version="4.7.1" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
This should probably not be used. Microsoft broke a great deal with this package, and it exists solely for legacy NET Frameworks that don't have better fixes. See [this issue](https://github.com/dotnet/runtime/issues/26131) for more details. It will likely cause downstream DLL issues for integrators.

I believe you can just use a framework reference for `net472` and `System.Net.Http` i.e.
``` csproj
  <ItemGroup Condition=" '$(TargetFramework)' == 'net472' ">
    <Reference Include="System.Net.Http" />
  </ItemGroup>
```
and I'm fairly sure (lacking a compiler to hand) that it won't be needed for `netstandard2.0`

Reply:
👍